lib/commands/general.js

```diff
@@ -204,8 +204,8 @@ commands.mobileBackdoor = async function mobileBackdoor (opts = {}) {
   if (target === 'element') {
     assertRequiredOptions(opts, ['elementId']);
   }
-  const {elementId} = opts;
-  return await this.espresso.jwproxy.command(`/appium/execute_mobile/backdoor`, 'POST', {target, methods, elementId});
+  const {elementId: targetElement} = opts;
+  return await this.espresso.jwproxy.command(`/appium/execute_mobile/backdoor`, 'POST', {target, methods, targetElement});
```

Comment:
wouldn't it be easier to fix it in the server code instead?

Reply:
No, this way is safer than changing the server code.
`MobileBackdoorParams` inherits `AppiumParams`. `AppiumParams` has `elementId` member. We should override `elementId` in `MobileBackdoorParams` (but the id in `AppiumParams` final.) I need to confirm and change the `AppiumParams`, too, then.